src/App.jsx

```diff
@@ -9,36 +11,17 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">
-              Tab 1
-            </a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">
-              Tab 2
-            </a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">
-              Tab 3
-            </a>
-          </li>
-        </ul>
-      </div>
+export const App = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```

Comment:
It would be nice to use a variable for the default value.
`const DEFAULT_VALUE = tabs[0]`
`const [selectedTab, setSelectedTab] = useState(tabs[0]);`

Reply:
Perhaps you're right.